executor/main_test.go

```diff
@@ -58,7 +60,26 @@ func createRequest(t *testing.T, filePath string, inputPaths []string) *http.Req
 	return req
 }
 
+func cleanUp() {
+	dir, err := ioutil.ReadDir("inputs")
+	if err != nil {
+		log.Error().
+			Err(err).
+			Msg("Error reading the contents of the 'inputs' directory")
+	}
+	for _, d := range dir {
```

Comment:
This is going to fail catastrophically if you don't return on the last error.


I would suggest passing `t *testing.T` and failing the test.